src/providers/data_providers/mockBusinessImageProvider.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseBusinessImageProvider } from './baseBusinessImageProvider';
+import { BusinessInfoInput } from '../../models/data_models/types';
+import { Image, ImageProviderName } from '../../models/data_models/image';
+
+export class MockBusinessImageProvider extends BaseBusinessImageProvider {
+	public async getBusinessImages(businessInfo: BusinessInfoInput): Promise<Array<Image>> {
+		return Promise.resolve([
```

Comment:
I might be wrong, but `Promise.resolve` might not be needed, since async functions always return a promise, and will implicitly wrap the return value in a promise if needed. If I'm wrong or if the intention is to be explicit, disregard this.